metron-interface/metron-alerts/src/app/shared/directives/alert-search.directive.ts

```diff
@@ -130,7 +130,7 @@ export class AlertSearchDirective implements AfterViewInit, OnChanges {
 
   private handleMouseEvent (callback: Function) {
     clearTimeout(this.mouseEventTimer);
-    this.mouseEventTimer = setTimeout(() => { callback(); }, 500);
+    this.mouseEventTimer = setTimeout(() => { callback(); }, 100);
```

Comment:
Was this change intentional?

Reply:
yeah i feel this is making some of the flaky tests stable, especially the ones where we hover on search text and delete them

Reply:
hey @iraghumitra - this slipped my mind earlier.  can you point me to the tests for this functionality?  I don't want this merged unless we have good tests.

Reply:
@nickwallen The status check was getting checked implicitly when we assert the table data. NowI added a explicit assert as well